sdks/python/apache_beam/runners/portability/portable_runner.py

```diff
@@ -141,23 +141,25 @@ def _create_environment(options):
           payload=(portable_options.environment_config.encode('ascii')
                    if portable_options.environment_config else None))
 
-  def default_job_server(self, options):
+  def default_job_server(self, portable_options):
     # TODO Provide a way to specify a container Docker URL
     # https://issues.apache.org/jira/browse/BEAM-6328
     if not self._dockerized_job_server:
       self._dockerized_job_server = job_server.StopOnExitJobServer(
           job_server.DockerizedJobServer())
     return self._dockerized_job_server
 
-  def create_job_service(self, options):
-    job_endpoint = options.view_as(PortableOptions).job_endpoint
+  def create_job_service(self, portable_options):
```

Comment:
I'd just leave this as `options`.

Reply:
I changed it because it is actually immediately converted back to portable_options. Basically the caller takes options and converts it to portable_options, then to the create_job_service method it passes options which is again converted to portable_options. It seems to be that we could pass portable_options in the first place and simplify the code a bit. However, I don't have a strong option, I'll revert back to options.